packages/mobile/src/components/core/IconButton.tsx

```diff
@@ -77,15 +77,9 @@ export const IconButton = ({
         activeOpacity={0.95}
         hitSlop={{ ...defaultHitSlop, ...hitSlop }}
       >
-        <View
-          style={[
-            styles.icon,
-            isDisabled && { opacity: 0.5 },
-            stylesProp?.icon
-          ]}
-        >
+        <View style={[styles.icon, stylesProp?.icon]}>
           <Icon
-            fill={fill}
```

Comment:
are there any buttons that rely on the 50% opacity for colors other than a lighter grey?

Reply:
Ah that is a good point, to be safe I'll add disabled fill as an option